packages/lightning-lsp-common/src/resources/sfdx/typings/lds.d.ts

```diff
@@ -2062,6 +2060,65 @@ declare module 'lightning/analyticsWaveApi' {
         sourceObjectFields: Array<string>;
     }
 
+    /**
+     * An Analytics template asset reference.
+     *
+     * https://developer.salesforce.com/docs/atlas.en-us.bi_dev_guide_rest.meta/bi_dev_guide_rest/bi_responses_asset_reference.htm
+     *
+     * Keys:
+     *    (none)
+     */
+    export interface TemplateAssetReferenceRepresentation {
+        /** The 18 character ID of the asset. */
+        id?: string;
+        /** The asset label. */
+        label?: string;
+        /** The asset developer name. */
+        name?: string;
+        /** The namespace that qualifies the asset name */
+        namespace?: string;
+        /** The asset URL. */
+        url?: string;
+    }
+
+    /**
+     * Representation for individual validation task
+     *
+     * https://developer.salesforce.com/docs/atlas.en-us.bi_dev_guide_rest.meta/bi_dev_guide_rest/bi_responses_template_readiness_item.htm
+     *
+     * Keys:
+     *    (none)
+     */
+    export interface TemplateReadinessItemRepresentation {
+        /** The icon/image associated with the validation task. */
+        image: TemplateAssetReferenceRepresentation | null;
+        /** The task specific label. */
+        label: string | null;
+        /** The task specific message. */
+        message: string | null;
+        /** The status for the readiness task */
+        readinessStatus: string | null;
+        /** The collection of tags describing the purpose of the validation. */
+        tags: Array<string>;
+        /** The task specific type, associated with readiness check type. */
+        type: string | null;
+    }
+
+    /**
+     * Representation for a single Wave Template verification.
+     *
+     * https://developer.salesforce.com/docs/atlas.en-us.bi_dev_guide_rest.meta/bi_dev_guide_rest/bi_responses_template_validate.htm
+     *
+     * Keys:
+     *    id (string): id
+     */
+    export interface TemplateValidateRepresentation {
+        /** The ID or fully qualified API name of this template. */
```

Comment:
Nit: is this comment right? It can be id or name?

Reply:
Yep. A number of our rest apis (and therefore adapters) support that in the urls and then the resulting output json.